src/components/autocomplete/autocomplete-field.tsx

```diff
@@ -24,6 +24,7 @@ export const AutocompleteField = ReactForm.FormField((props: AutocompleteProps &
             renderInput={(inputProps) => (
                 <input
                     {...inputProps}
+                    qe-id={props.qeId}
```

Comment:
minor - I'm pretty sure this needs to be camel-case.

Reply:
Hi @alexec, this is a 'custom' attribute for the `input` element. I tried camel-case but it's an error because the property does not exist on type 'DetailedHTMLProps<InputHTMLAttributes<HTMLInputElement>, HTMLInputElement>'.ts(2322).

BTW. I had to make some additional changes.  I added ids to the OK and Cancel buttons in popup-manager.tsx.  (So that we can delete the app and dismiss the confirmation popup).

One other change is to the `qeId` property in autocomplete.tsx, which is now `lowercase` qeid.  This property is referenced in the FormComponent element in form-field.tsx.  In that case `qeid` is passed as a property to the AutoCompleteField component (autocomplete-field.tsx) but it _also_ can be an attribute to a React component, such as **Text**.  This is the only way I can get the form-fields to work so that there are no React errors or warnings.